### DISCUSSION TITLE
About save filtered task queues

### DISCUSSION THREAD

**Discussion by jc3213** [jc3213/download_with_aria2#70] [Polls]
I will add save filtered task queues function to `Task Manager`. I wonder if I'll add `Save` and `Default` button to `Task Queue` menu or just apply the changes when they are done.

There's a preview build with buttons - no function yet
[popup_v1.zip](https://github.com/jc3213/download_with_aria2/files/11950028/popup.zip)

Here are screenshots
<details>
<summary>Popup</summary>
<img src="https://github.com/jc3213/download_with_aria2/assets/8744936/f41118c2-47a0-4d5e-a92e-5a992cb0cd81"></img>
</details>
<details>
<summary>Open in Tab</summary>
<img src="https://github.com/jc3213/download_with_aria2/assets/8744936/34038a2f-3d08-4e57-9da6-81f476193887"></img>
</details>

**Comment by jc3213** [comment]
@ivysrono @daiaji @wordpure 帮忙看看？

**Reply by daiaji** [reply]
沉迷用LLM和黄油打胶，可能暂时搞不了呢😶

**Reply by jc3213** [reply]
能炼LLM的大佬，今年显卡太贵，我还在1060坚持

**Reply by ivysrono** [reply]
现在炼丹起步要什么级别显卡？

**Reply by ivysrono** [reply]
感觉你保持统一就好，选项调整有保存按钮，那么就都有。

**Reply by jc3213** [reply]
> 现在炼丹起步要什么级别显卡？

炼丹看性能，但是LLM跟不少模型要足够大的显存去加载。目前最水桶的是4090，性价比选择就是2080 Ti 22GB，有钱直接H100 一步到位。
